{
  "createdAt": "2023-01-07T08:51:10.161Z",
  "updatedAt": "2023-10-23T01:20:35.000Z",
  "id": "4",
  "name": "Ошибка",
  "active": false,
  "nodes": [
    {
      "parameters": {},
      "id": "b0b7723f-dac5-41b4-ac2d-ec283146406e",
      "name": "Ошибка",
      "type": "n8n-nodes-base.errorTrigger",
      "typeVersion": 1,
      "position": [
        360,
        520
      ]
    },
    {
      "parameters": {
        "chatId": "198424221",
        "text": "=Ошибка в задаче <a href=\"{{ $json.execution.url }}\">{{ $json.workflow.name }}</a>\n{{ $json.execution.error.message }}\nStacktrace:\n{{ $json.execution.error.stack }}",
        "additionalFields": {
          "parse_mode": "HTML"
        }
      },
      "id": "e16eb7c5-cd9f-44a0-a8e5-e37ed4b0ba99",
      "name": "Сообщаем в ТГ",
      "type": "n8n-nodes-base.telegram",
      "typeVersion": 1,
      "position": [
        600,
        520
      ],
      "credentials": {
        "telegramApi": {
          "id": "6",
          "name": "Ассистент"
        }
      }
    }
  ],
  "connections": {
    "Ошибка": {
      "main": [
        [
          {
            "node": "Сообщаем в ТГ",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "settings": {
    "callerPolicy": "workflowsFromSameOwner",
    "saveManualExecutions": true,
    "executionOrder": "v1",
    "errorWorkflow": "86jQEQoIzxdPTWAz"
  },
  "staticData": null,
  "meta": null,
  "pinData": {},
  "versionId": "922ec09a-c5c0-4b02-b386-ab6559ac0d73",
  "triggerCount": 0,
  "tags": []
}